src/App.jsx

```diff
@@ -14,57 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const startIndex = goods.findIndex(products => products === 'Jam');
+  const [activeRow, setActiveRow] = useState(startIndex);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const [message, setMessage] = useState('Jam');
+  const [isButtonVisible, setIsButtonVisible] = useState(true);
+  const RemoveButton = () => {
+    setMessage('No goods');
+    setActiveRow(null);
+    setIsButtonVisible(false);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {isButtonVisible ? (
+        <h1 className="title is-flex is-align-items-center">
+          {message} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              RemoveButton();
+            }}
+          />
+        </h1>
+      ) : (
+        <h1 className="title">{message} selected</h1>
```

Comment:
use here full phare here because message will be empty:

`<h1 className="title">No goods selected</h1>`